### DISCUSSION TITLE
Standalone use (diy rack mounted fans controller)

### DISCUSSION THREAD

**Discussion by stvnjnk** [tjko/fanpico#35] [Ideas]
Was looking for a pwm fan controller for my noctua fans in my rack and came across this project (10/10). Can the board handle the fans power with just the aux 4pin connector and could it be connected to a raspberry pi as a standalone device without the 4 motherboard output connectors. Thanks for the help

**Comment by tjko** [comment]
Yes, I've used the board myself with just 4-pin "Floppy" power connector fine. For the AUX power connector, there is 3A poly fuse though, but that should not be a problem unless using some "industrial" fans, etc..

If there is no PWM signal inputs from a "motherboard", fans can be controlled using Thermocouples (or just the Pico internal temperature sensor).  Alternatively a program (or script) could adjust the fan speed by issuing "CONF:FANx:SOURCE FIXED,n" commands (where x is the fan number and n is the duty cycle 0..100) over the USB (serial) console connection.

https://github.com/tjko/fanpico/blob/main/commands.md#configurefanxsource
